python/cudf/cudf/core/column/column.py

```diff
@@ -38,7 +39,7 @@
 from cudf.utils.utils import buffers_from_pyarrow, mask_dtype
 
 
-class ColumnBase(Column):
+class ColumnBase(Serializable, Column):
```

Comment:
Do we explicitly want the MRO to be `ColumnBase, Serializable, Column` instead of `ColumnBase, Column, Serializable`?

Reply:
This question applies for all multi-inheritance below as well.

Reply:
Are you just wanting to swap the order in the parens? If so, I have no strong feelings. Happy to swap them if you prefer.

Reply:
Yup - let's do that. Thanks, John -- I really love how much cleaner this abstraction is!

Reply:
Thanks, have updated accordingly.